### PR TITLE
[ES] Update some `People` articles

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- `SKIP_OUTDATED_CHECK`: a lowercase
- `SKIP_WIKILINK_CHECK`: wiki link pointing to an untranslated article

### CODE REVIEW

File: wiki/Main_page/es.md [hunk 1]
```diff
@@ -1,9 +1,3 @@
----
-layout: main_page
```
Comment (The-Last-Cookie) [inline wiki/Main_page/es.md:2]: I think the `layout: main_page` tag should not be removed.

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]

File: wiki/Main_page/es.md (hunk 1 above)
Comment (agatemosu) [reply wiki/Main_page/es.md:2]: ahshajs thanks

**Comment by agatemosu** [comment]
@UnknownOsu está bien el PR?

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Comment by agatemosu** [comment]
sorry the last cookie miss click asjdkjk

**Comment by The-Last-Cookie** [comment]
No worries! I would be glad to help in reviewing formatting though.

**Review by TicClick — APPROVED** [review]

**Pull request merged**